jsk_aero_robot/jsk_aero_startup/package.xml

```diff
@@ -16,5 +16,10 @@
   <run_depend>aeroeus</run_depend>
   <run_depend>jsk_maps</run_depend>
   <run_depend>jsk_robot_startup</run_depend>
-
+  <run_depend>linux_kinect</run_depend>
```

Comment:
what is this????

Reply:
This package is located here: https://github.com/sasabot/win_lin_kinect
Bridge to bringing kinect functions working on windows into linux.